src/Components/test/testassets/Directory.Build.props

```diff
@@ -0,0 +1,3 @@
+<Project>
+  <Import Project="..\..\Blazor\Directory.Build.props" />
+</Project>
```

Comment:
Looks like we're getting some complicated dependencies between different parts of the source tree. Do you consider this temporary? Should we be looking to refactor to avoid this weirdness, or is this not going into `master`?

Reply:
This isn't going into master, it's an artifact of having to deal with the shared framework